src/react-components/info-dialog.js

```diff
@@ -217,6 +218,26 @@ class InfoDialog extends Component {
           </div>
         );
         break;
+      case InfoDialog.dialogTypes.webvr_recommend:
+        dialogTitle = "Enter in VR";
+        dialogBody = (
+          <span>
+            To enter Hubs with Oculus or SteamVR, you can use Firefox.
+            <p />
+            <a className="info-dialog--action-button" href="https://www.mozilla.org/firefox">
+              Download Firefox
+            </a>
+            <p />
+            <span style={{ fontSize: "0.8em" }}>
```

Comment:
can you use a `<p>…</p>`? they are not self-closing tags and the `<span>` adds more weight to the markup, bundle, DOM, etc.